jax_unirep/evotuning.py

```diff
@@ -136,8 +160,29 @@ def length_batch_input_outputs(
     return xs, ys
 
 
+def get_batch_len(batched_seqs):
```

Comment:
@ivanjayapurna, just wondering, do you have a software test added for this function?

If not, can we add one inside the `test_evotuning.py` test suite? An example-based test is sufficient. Here's one to get you started.

```python
def test_get_batch_len():
    batched_seqs = [["ABC", "ACD"], ["AABC", "EKQJ"], ["QWLRJK", "QJEFLK"]]
    mean_batch_length, batch_lengths = get_batch_len(batched_seqs)
    assert mean_batch_length == 2
    assert batch_lens = [2, 2, 2]
```